lib/config.js

```diff
@@ -0,0 +1,246 @@
+"use strict";
+/** Module to define plugin configuration and react on change. */
+var logger_1 = require('./logger');
+var CompositeDisposable = require('atom').CompositeDisposable;
+var logger = logger_1.Logger.getInstance();
+/** Plugin configuration visible in Atom editor */
+exports.PluginConfig = {
```

Comment:
If you move your configuration to a `configSchema` section in your `package.json` Atom can show your settings without having to transpile+load+run your code.

You can see an example [here](https://github.com/AtomLinter/linter-stylelint/blob/master/package.json#L33-L51), note that this requires Atom v1.4.0 minimum.